third_party/ijar/test/ijar_test.sh

```diff
@@ -114,13 +116,49 @@ function check_consistent_file_contents() {
 
 function set_up() {
   mkdir -p $TEST_TMPDIR/classes
+  mkdir -p $TEST_TMPDIR/kotlin
 }
 
 function tear_down() {
   rm -fr $TEST_TMPDIR/classes
+  rm -fr $TEST_TMPDIR/kotlin
+}
+
+function run_ijar_and_extract_to() {
+    mkdir -p $3 || fail "could not ensure output dir"
+    ${IJAR} $1 $2 || fail "ijar failed"
+    pushd $3
+        ${JAR} xf $2 || fail "could not extract jar contents"
```

Comment:
Please just indent with 4 spaces here, too.

Reply:
It's showing up as 4 spaces on my side (not a tab).